eng/native/version/copy_version_files.sh

```diff
@@ -0,0 +1,21 @@
+__VersionFolder="$(cd "$(dirname "$0")"; pwd -P)"
+__RepoRoot="$(cd "$(dirname "$__VersionFolder")/../../"; pwd -P)"
+
+for path in "${__VersionFolder}/"*{.h,.c}; do
+    if [[ "$(basename $path)" == _version.c ]]; then
+        # For _version.c, update the commit ID if it has changed from the last build.
+        # Set IFS to nothing to prevent the shell from combining all of the piped output into a single line in the script below
+        IFS=
```

Comment:
Just out of curiosity, is it `cat` jumbling the lines?

Reply:
I think it's bash jumbling the lines that are outputted from cat and sed.